generate-manifest.py

```diff
@@ -62,7 +62,6 @@ def edit_manifest(data, arch, branch, runtime_version):
     # This is nasty
     gtk_patches = [
         'gtk3-fix-atk-gjs-crash.patch',
```

Comment:
We should really try to shepherd this one into upstream, but that's out of scope for this PR.